packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/hooks/useOutboundLineEditRows.ts

```diff
@@ -0,0 +1,89 @@
+import { useEffect, useMemo } from 'react';
+import { useTableStore, SortUtils } from '@openmsupply-client/common';
+import { DraftOutboundLine } from '../../../../types';
+import { PackSizeController } from '../hooks';
+import { isA } from '../../../../utils';
+
+export const useOutboundLineEditRows = (
+  rows: DraftOutboundLine[],
+  packSizeController: PackSizeController
+) => {
+  const tableStore = useTableStore();
+
+  const {
```

Comment:
I had the same kinda thing - just took this in entirety as I though merging would be easier that way.

Reply:
I think I'll just close my PR?

Reply:
yeah, might as well. I think I've got everything in here now